### ISSUE TITLE
Invisible Entities

### ISSUE THREAD

**Issue by Ryudragon2** [sciguyryan/TheCrackPack#910]
Hello my Friend is having problem he cannot see the biomes o'plenty and he also doesn't see the sludge from minecraft reloaded is there anyone that can help

**Comment by evknucklehead** [comment]
The first thing to check is does he have the latest drivers available for his video card? Followed by, Is he using a Resource Pack?

Another thing to check is are you sure he has the same version of the pack as you and the same version as the server you're using?

**Comment by Ryudragon2** [comment]
its fixed i had to reinstall the server

**Comment by evknucklehead** [comment]
Yeah, sorry about not responding sooner. I'm currently on a road trip with my grandfather, and today's the first time in a few days I've been able to get online.

**Comment by Ryudragon2** [comment]
its cool you might wanna check the particals for the taint swarm
